### PR TITLE
MAISTRA-2209: temporarily add perms to list/watch namespaces for istiod

### PR DESCRIPTION
**Pull request by rcernich**
Also update the charts from maistra/istio.

### CODE REVIEW

File: resources/helm/overlays/istio-control/istio-discovery/templates/meshclusterrole.yaml [hunk 1]
```diff
@@ -11,3 +11,7 @@ rules:
   - apiGroups: ["authentication.k8s.io"]
     resources: ["tokenreviews"]
     verbs: ["create"]
+  # XXX: remove before 2.1 release
```
Comment (jwendell) [inline resources/helm/overlays/istio-control/istio-discovery/templates/meshclusterrole.yaml:14]: Link the Jira issue here, and link this line in the jira issue :)

**Review by brian-avery — APPROVED** [review]

File: resources/helm/overlays/istio-control/istio-discovery/templates/meshclusterrole.yaml (hunk 1 above)
Comment (rcernich) [reply resources/helm/overlays/istio-control/istio-discovery/templates/meshclusterrole.yaml:14]: done

**Review by jwendell — APPROVED** [review]
I fixed the PR title

File: resources/helm/overlays/istio-control/istio-discovery/templates/meshclusterrole.yaml (hunk 1 above)
Comment (dgn) [reply resources/helm/overlays/istio-control/istio-discovery/templates/meshclusterrole.yaml:14]: wasn't this solved by https://github.com/maistra/istio/pull/283?

Comment (rcernich) [reply resources/helm/overlays/istio-control/istio-discovery/templates/meshclusterrole.yaml:14]: Unfortunately, no. :(

Comment (dgn) [reply resources/helm/overlays/istio-control/istio-discovery/templates/meshclusterrole.yaml:14]: ahh got it, only now read the issue comments. thanks!

**Pull request merged**